monai/visualize/img2tensorboard.py

```diff
@@ -67,31 +79,37 @@ def make_animated_gif_summary(
     axis_order = [0] + animation_axes + image_axes
 
     slicing = []
-    for i in range(len(tensor.shape)):
+    for i in range(len(image.shape)):
         if i in axis_order:
             slicing.append(slice(None))
         else:
             other_ind = other_indices.get(i, 0)
             slicing.append(slice(other_ind, other_ind + 1))
-    tensor = tensor[tuple(slicing)]
+    image = torch.as_tensor(image[tuple(slicing)])
```

Comment:
If the `image` is numpy array, maybe we need to call `np.ascontguousarray()` as `torch.as_tensor()` needs?

Reply:
Testing a slightly different approach to accomplish the same task.